### PR TITLE
test cleanup

### PR DESCRIPTION
**Pull request by sfraczek**
few fixes to tests

### CODE REVIEW

File: doc/source/provider_video.rst [hunk 1]
```diff
@@ -98,7 +98,7 @@ appropriate entry for video options:
     augmentation_config = {"type": "image",
                            "scale": [0.875, 0.875]}
 
-    aeon_config = {"manifest_filename": "train.csv",
+    aeon_config = {"manifest_filename": "train.tsv",
                    "etl": (video_config, label_config),
```
Comment (AdrShv) [inline doc/source/provider_video.rst:102]: why ? tsv we had becouse of typo in benchmark test

**Review by AdrShv — CHANGES_REQUESTED** [review]

File: doc/source/provider_video.rst (hunk 1 above)
Comment (sfraczek) [reply doc/source/provider_video.rst:102]: We have some places using csv and some tsv. CSV suggests commas as separators and is a widely known format, but we use tabs everywhere as separators so tsv made more sense to me.

Comment (sfraczek) [reply doc/source/provider_video.rst]: We have some places using csv and some tsv. CSV suggests commas as separators and is a widely known format, but we use tabs everywhere as separators so tsv made more sense to me.

Comment (AdrShv) [reply doc/source/provider_video.rst:102]: ok, you are right

**Review by AdrShv — APPROVED** [review]

**Review by kbinias — APPROVED** [review]
LGTM, in new PR we should add train-index.tsv and val-indes.tsv for ImageNet to release version of aeon.

**Pull request merged**